app/src/views/private/components/revisions-drawer-detail/revisions-drawer-updates.vue

```diff
@@ -61,7 +61,7 @@ export default defineComponent({
 					const currentValue = props.revision.delta[fieldKey];
 					const previousValue = previousRevision.value.data[fieldKey];
 
-					if (isEqual(currentValue, previousValue)) return null;
+					if (isEqual(currentValue, previousValue) && currentValue !== '**********') return null;
```

Comment:
I don't think we should rely on the arbitrary string value `******` for the conceal check.  It's not unthinkable that the redaction string is going to be configurable in the future, which would cause issues here.

Reply:
> I think we would still require a check if a string is concealed.

We should pull that information from the field store than, not the returned value 👍🏻

Reply:
Oh yes! Done. Thanks!!